.github/workflows/pre-submit.check-dist-sha256.yml

```diff
@@ -0,0 +1,48 @@
+name: Check dist sha256
```

Comment:
Could you move this code into the existing `pre-submit.actions.yml` file?

Reply:
Can this wait for a follow-up PR? These aren't blockers.

Reply:
I have moved it to the other file.